firestore/android/FirestoreSnippetsCpp/app/src/main/cpp/snippets.cpp

```diff
@@ -1061,6 +1112,54 @@ void ReadDataPaginateQuery(firebase::firestore::Firestore* db) {
   // [END paginate]
 }
 
+// https://firebase.google.com/docs/firestore/bundles#loading_data_bundles_in_the_client
+void LoadFirestoreBundles(firebase::firestore::Firestore* db) {
+  using firebase::Future;
+  using firebase::firestore::Error;
+  using firebase::firestore::Firestore;
+  using firebase::firestore::LoadBundleTaskProgress;
+  using firebase::firestore::Query;
+  using firebase::firestore::QuerySnapshot;
+
+  // [START bundled_query]
+  db->LoadBundle("bundle_name", [](const LoadBundleTaskProgress& progress) {
+    switch(progress.state()) {
+      case LoadBundleTaskProgress::State::kError: {
+        // The bundle load has errored. Handle the error in the returned future.
+        return;
+      }
+      case LoadBundleTaskProgress::State::kInProgress: {
+        std::cout << "Bytes loaded from bundle: " << progress.bytes_loaded()
+                  << '\n';
```

Comment:
nit: use `std::endl` instead of `'\n'` for portability. Applies elsewhere in this file too.

Reply:
Done.